src/gpx/generator/utils/route/format_points.py

```
@@ -0,0 +1,5 @@
+def format_points(args):
```

Ruff (0.8.2)

No such file or directory (os error 2)

(E902)